homeassistant/components/elgato/light.py

```diff
@@ -144,3 +155,10 @@ def device_info(self) -> DeviceInfo:
             "model": self._info.product_name,
             "sw_version": f"{self._info.firmware_version} ({self._info.firmware_build_number})",
         }
+
+    async def async_identify(self) -> None:
+        """Identify the light, will make it blink."""
+        try:
+            await self.elgato.identify()
+        except ElgatoError:
```

Comment:
Should this mark the elgato light as unavailable?

Reply:
I was not sure about that... I guess it won't harm 👍

Reply:
Done.